comptests/ha/test_restart_pod.py

```diff
@@ -318,8 +542,177 @@ def test_serverpod_status_online_after_replicas(self):
+        self.deployment_name = deployment_name_ha = resp[1]
+        self.restore_method = common_const.RESTORE_SCALE_REPLICAS
+        self.restore_pod = True
+
+        LOGGER.info("Step 2: Check delete pod alert in HA logs.")
+        for log in common_const.HA_SHUTDOWN_LOGS:
+            resp = self.ha_comp_obj.check_string_in_log_file(node_obj, "SIGTERM", log)
+            assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 2:Verified all HA logs for delete pod alert message")
+
+        LOGGER.info("Step 3: Shutdown the data pod by making replicas=0")
+        LOGGER.info("Get pod name to be deleted")
+        pod_list = self.node_master_list[0].get_all_pods(pod_prefix=common_const.POD_NAME_PREFIX)
+        pod_name = random.sample(pod_list, 1)[0]
+        LOGGER.info("Deleting pod %s", pod_name)
+        resp = self.node_master_list[0].create_pod_replicas(num_replica=0, pod_name=pod_name)
+        assert_utils.assert_false(resp[0], f"Failed to delete pod {pod_name} by making replicas=0")
+        LOGGER.info("Step 3: Successfully shutdown/deleted pod %s by making replicas=0", pod_name)
+        self.deployment_name = deployment_name_data = resp[1]
+        self.restore_method = common_const.RESTORE_SCALE_REPLICAS
+
+        LOGGER.info("Step 4: Start data pod by making replicas=1")
+        resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                       restore_method=self.restore_method,
+                                       restore_params={"deployment_name": deployment_name_data})
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
+        LOGGER.info("Step 4: Successfully started the data pod again by making replicas=1")
+
+        LOGGER.info("Step 5: Start HA pod by making replicas=1")
+        resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                       restore_method=self.restore_method,
+                                       restore_params={"deployment_name": deployment_name_ha})
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
+        LOGGER.info("Step 5: Successfully started the ha pod again by making replicas=1")
+        self.restore_pod = False
+
+        LOGGER.info("Step 6: Verify publish action for event sent to component Hare")
+        resp = self.ha_comp_obj.check_string_in_log_file(node_obj, "to component hare",
+                                                         common_const.HA_SHUTDOWN_LOGS[2], lines=4)
+        assert_utils.assert_true(resp[0], "Alert not sent to hare")
+        LOGGER.info("Step 6: Successfully verified publish action for event sent to component Hare")
+
+    @pytest.mark.comp_ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-36177")
+    def test_hapod_status_offline_after_delete_deployment(self):
```

Comment:
We are not going to automate these scenario because of some design change. Please remove this.